Makefile

```
@@ -18,15 +19,17 @@ VENV ?= .venv3
 
 all: clean images tests
 
-install: .install
+install: .install .images .env
```

Might be better if we just have it in the docs to copy to `.env` (or run `make env`). As we still have to enter in the RHSM values
```suggestion
install: .install .images
```

---

I tried to solve it with the single command  - make install.
And some values of the secrets might have values (even they could be different for upstream and downstream)

---

could we leave it as it is for now?